packages/terra-form-select/CHANGELOG.md

```diff
@@ -4,10 +4,16 @@ ChangeLog
 Unreleased
 ----------
 ### Added
+* Added Combobox and ComboboxField exports and doc site entries
+* Added DropdownSelect and DropdownSelectField exports and doc site entries
+* Added MultipleSelect and MultipleSelectField exports and doc site entries
+* Added SearchSelect and SearchSelectField exports and doc site entries
+* Added TagSelect and TagSelectField exports and doc site entries
```

Comment:
Similar to Combobox, what are peoples thoughts on dropping the Select suffix. `Search` and `Tag` respectively? Search may have a little confusion with the existing SearchField, but `SearchSelect` and `TagSelect` feels really strange, especially with Combobox not following the same pattern.

Reply:
Naming is hard.

Just like the confusion with `SearchField`, `Tag` sounds like it's just the `_Tag` component (pill, chip, tag, whatever it's called) which multi-selected values are displayed as.

Reply:
Spoke offline and the consensus was to keep these as SearchSelect and TagSelect